at_secondary/at_persistence_secondary_server/lib/src/notification/at_notification_keystore.dart

```diff
@@ -81,20 +83,55 @@ class AtNotificationKeystore
       bool? isBinary,
       bool? isEncrypted,
       String? dataSignature}) async {
-    // TODO: implement deleteExpiredKeys
     throw UnimplementedError();
   }
 
   @override
   Future<bool> deleteExpiredKeys() async {
-    // TODO: implement deleteExpiredKeys
-    return Future.value(false);
+    var result = true;
```

Comment:
Stats for notifications around expiry needs to be exposed. Lets address through a different ticket.

Reply:
sure